src/main/kotlin/model/BookVideoLink.kt

```diff
@@ -92,6 +92,7 @@ class BookVideoLink private constructor() {
 
       views.createDeploymentView(system, "bookVideoLinkContainerProductionDeployment", "The Production deployment scenario for the Book Video Link service").apply {
         addDefaultElements()
+        add(AWS.london)
```

Comment:
this is curious as it should be automatic after any `CloudPlatform.kubernetes.add(this)`

Reply:
I didn't remove the default elements from this view - have done this now.